snow/networking/handler/health.go

```diff
@@ -54,5 +57,24 @@ func (h *handler) networkHealthCheck() (interface{}, error) {
 		)
 	}
 
+	details["disconnectedValidators"] = h.getDisconnectedValidators()
```

Comment:
this will always return disconnectedValidators in the health details message. if we want we can move it to above and return only when it's under the minPercentConnected

Reply:
Sorry I did not understand this, I directly set it to details without requiring any check for minimum connected. 

At first I intended to move this inside the above if clause, so we only show it when it errors. Currently it shows regardless of if minConnectedPercent>connectedPercent or not (to be a bit more verbose). If we think this can be too noisy, I can move it.